homeassistant/components/media_player/plex.py

```diff
@@ -296,30 +311,17 @@ def __init__(self, config, device, session, plex_sessions,
                         'media_player', prefix,
                         self.name.lower().replace('-', '_'))
 
-    def _clear_media(self):
+    def _clear_media_details(self):
         """Set all Media Items to None."""
-        # General
-        self._media_content_id = None
-        self._media_content_rating = None
-        self._media_content_type = None
-        self._media_duration = None
-        self._media_image_url = None
-        self._media_title = None
-        self._media_position = None
-        # Music
-        self._media_album_artist = None
-        self._media_album_name = None
-        self._media_artist = None
-        self._media_track = None
-        # TV Show
-        self._media_episode = None
-        self._media_season = None
-        self._media_series_title = None
+        for media_var in filter(lambda x: x.startswith('_media_'), dir(self)):
```

Comment:
Don't make this change. It obfuscates what's happening, isn't greppable, and has potential for nasty unintended side-effects.

Reply:
I actually did that as i really hated the "warnings" i was getting about the media stuff being declared outside of __init__ and i thought it was neater.. but yes looking at it it does make things more fragile and less obvious.